docs/downstream-integrations/api/authentication.md

```diff
@@ -24,22 +24,133 @@ The AE token must be passed in the HTTP `Authorization` (case insensitive) reque
 Authorization: AE <ae-token>
 ```
 
-The AE token is generated on the server when a user has successfully signed in with Collab.Land. The AE token is valid for 1 hour after which it expires and will need to be refreshed.
+The AE token is generated on the server when a user has successfully signed in with Collab.Land. The AE token is valid for 1 hour after which it expires and will need to be refreshed. -->
 
 ## Login with Collab.Land (LWC)
 
-The LWC flow process offers a way for client applications to authenticate their users through Collab.Land. Applications registered with Collab.Land can redirect users to `https://login.collab.land` to authenticate themselves with Collab.Land and grant permissions to the client application for requested resources.
+The Collab.Land API OAuth2 flow provides a secure and standardized method for client applications to authenticate users and access resources on Collab.Land. The flow follows the OAuth2 protocol, allowing applications to obtain an access token that can be used to interact with the Collab.Land API on behalf of the authenticated user.
+
+## Register your OAuth2 Client Application
+
+To begin using the Collab.Land API OAuth2 flow, you must first register your client application via the [Collab.Land Developer Portal](https://dev-portal.collab.land/). After the registration process, you will receive the following credentials:
+
+- **Client ID**: A unique identifier for your client application.
+- **Client Secret**: A confidential string used to authenticate your client application with Collab.Land.
+- **API Key**: A secret key used to authenticate your client application with Collab.Land.
```

Comment:
Please note that `API key` does not play any role in oAuth2 flows. It can be used as a convenient way to invoke Collab.Land APIs that don't involve any users, such as token-gating services.

API key is basically a pre-generated access token for the given client application but it doesn't have the control such as expiration or scopes. In the future, we may require oAuth client credentials flow for such use cases.